frontend/src/pages/workspace/profile/Index.tsx

```diff
@@ -0,0 +1,111 @@
+import { Container, Stack, Avatar, Typography, Button, FormControl } from "@mui/material";
+import { TextFieldElement, FormContainer } from "react-hook-form-mui";
+import { useSelector } from "react-redux";
+import { selectUser } from "../../../store/userSlice";
+import { useEffect, useMemo, useState } from "react";
+import { useDebounce } from "react-use";
+import { useUpdateUserNicknmaeMutation } from "../../../hooks/api/user";
+import { useCheckNameConflictQuery } from "../../../hooks/api/check";
+import { useGetWorkspaceQuery } from "../../../hooks/api/workspace";
+import { useParams } from "react-router-dom";
+const avatarSize = 117;
+
+function ProfileIndex() {
+	const params = useParams();
+	// eslint-disable-next-line
+	const { data: workspace } = useGetWorkspaceQuery(params.workspaceSlug);
+
+	const userStore = useSelector(selectUser);
+	const [nickname, setNickname] = useState(userStore.data?.nickname || "");
+	const [debouncedNickname, setDebouncedNickname] = useState("");
+	const { data: conflictResult } = useCheckNameConflictQuery(debouncedNickname);
+	const { mutateAsync: updateUserNickname } = useUpdateUserNicknmaeMutation();
+	const errorMessage = useMemo(() => {
+		if (conflictResult?.conflict) {
+			return "Already Exists";
+		}
+		return null;
+	}, [conflictResult?.conflict]);
+
+	const isSumbit = () => {
+		return (
+			Boolean(errorMessage) || nickname === userStore.data?.nickname || nickname.length === 0
+		);
+	};
+
+	useDebounce(
+		() => {
+			setDebouncedNickname(nickname);
+		},
+		500,
+		[nickname]
+	);
+
+	useEffect(() => {
+		if (userStore.data?.nickname) {
+			setNickname(userStore.data?.nickname || "");
+		}
+	}, [userStore.data?.nickname]);
+
+	const handleNicknameChange = (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+		setNickname(e.target.value);
+	};
+
+	const handleUpdateUserNickname = async (data: { nickname: string }) => {
+		await updateUserNickname(data);
+	};
+
+	return (
+		<Container sx={{ height: "calc(100vh - 88px)", width: "100%" }}>
+			<Stack alignItems="center" justifyContent="center" gap={6} sx={{ height: 1 }}>
+				<Avatar
+					sx={{
+						width: avatarSize,
+						height: avatarSize,
+						fontSize: avatarSize / 2,
+					}}
+				>
+					{userStore.data?.nickname?.charAt(0).toUpperCase()}
+				</Avatar>
+				<Stack width={310}>
+					<FormControl>
+						<Typography variant="body1">User name</Typography>
+						{userStore.data?.nickname && (
+							<FormContainer
+								defaultValues={{ nickname: userStore.data?.nickname }}
+								onSuccess={handleUpdateUserNickname}
+							>
+								<Stack gap={3}>
+									<TextFieldElement
+										variant="standard"
+										name="nickname"
+										value={nickname}
```

Comment:
I think we can delete this line.
Because `FormControl` will use `defaultValues`.

Refer to `frontend/src/components/modals/ChangeNicknameModal.tsx`.

Reply:
thank you. 

i deleted unused value

[5ffb99e](https://github.com/yorkie-team/codepair/pull/237/commits/5ffb99ede13363feff8d7a322abe3bd96608158b)